examples/annotation_import/image.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    import uuid
</pre><p>not really your change, but you might want to consider cleaning up imports. uuid is a duplicate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sounds good </p>